### DISCUSSION TITLE
Task 2 Submission

### DISCUSSION THREAD

**Discussion by DerryLin-Tamu** [SimDynamX/SpaceTeamsPRO#72] [2024 Lunar SAR Competition Q&A] [answered]
Has task 2 submission link has been uploaded yet?

**Comment by SkylerHawkins** [comment]
I was wondering the same thing. I read through all emails they sent out and watched linked videos, but so far have found nothing in terms of directions/submission portals.

**Comment by rcghpge** [comment]
bro give us more time lol yo any1 wanna connect hmu gg's

**Comment by connorjak** [comment, marked as answer]
We haven't posted the form yet, sorry. Form will be posted and deadline will be delayed. We're all at a conference this weekend.

**Comment by DerryLin-Tamu** [comment]
https://docs.google.com/forms/d/e/1FAIpQLScGXPcFGRbH2pcMkVmPEhx9PAgpI9Trq7ACmmMMiLoqH6pe7w/viewform
